augur/augur_client.py

```diff
@@ -250,9 +245,10 @@ def load_market_id_from_order_id(self, order_id):
         self._contract_abi)
       self._event_signature_to_name_map = inverse_dict(
         self._event_name_to_signature_map)
+      self._latest_block = self._ethereum_client.eth.blockNumber
```

Comment:
What is ```_latest_block``` used for?

Reply:
`_latest_block` holds the latest Ethereum block at the time of the AugurClient startup. This was initially used to hold the end block for the historical listener, but does not do anything at the moment in the current branch as we supply the end block to `filter_blocks()`.